app/helpers/api/bapi/getSelectedPrice.js

```diff
@@ -0,0 +1,11 @@
+import { getData } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { getEndpoint } from '../../../endpoints';
+
+export const getSelectedPrice = async ({ selectedPriceId, accessToken }) => {
```

Comment:
Just a suggestion but the context of this function is really about retrieving a single price. BAPI doesn't know anything about a "selected" price, so could be a little misleading. Very minor so please don't feel like you have to make the change.

Reply:
😂  I was also thinking the same thing. I'm going to leave this name as is for now. Since I want to refactor the other ones to use this too. So finding where we have used `getSelectedPrice` will be easier later on.